read.go

```diff
@@ -321,7 +320,7 @@ func (r *reader) readIndexData(toc *indexTOC) (*indexData, error) {
 
 	d.fileNameIndex = toc.fileNames.relativeIndex()
 
-	d.fileNameNgrams, err = d.readFileNameNgrams(toc)
+	d.fileNameNgrams, err = d.newBtreeIndex(toc.nameNgramText, toc.namePostings)
```

Comment:
I guess before this leaves draft this will be upgraded to a feature flag?

Reply:
yep. The PR wasn't ready for review. I was still in process of making it work.